.github/workflows/ci-rust.yml

```diff
@@ -146,13 +147,30 @@ jobs:
         run: echo OPENSSL_SRC_PERL=C:/Strawberry/perl/bin/perl >> $GITHUB_ENV
         timeout-minutes: 1
 
+      - name: Install winfsp
+        if: startsWith(matrix.os, 'windows')
+        shell: bash
+        run: |
+          set -eux
+          choco install winfsp -y --version=${{ env.WINFSP_VERSION }}
+          mkdir -vp ${{ runner.temp }}
```

Comment:
```suggestion
```

The temporary folder is already present, no need to create it

Reply:
Ty

Reply:
Something goes wrong, because D:\a\_temp exists, but ${{ runner.temp }} is D:\a_temp :/ @FirelightFlagboy